### PR TITLE
internal_dev_guide.md: create a custom Python distribution

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

File: internal_dev_guide.md [hunk 1]
```diff
@@ -5,52 +5,48 @@ Internal dev guide (for AdaCore developers)
...
+git clone git@github.com:AdaCore/langkit-query-language.git
+cd langkit-query-language
 ```
 
-2. Checkout LKQL
+3. Install Python dependencies via `pip`:
 
 ```sh
-git clone git@github.com:AdaCore/langkit-query-language.git
+pip3 install -r REQUIREMENTS.txt
```
Comment (leogermond) [inline internal_dev_guide.md:21]: The requirements.txt file has not been added.

Comment (leogermond) [reply internal_dev_guide.md:21]: And it would be nice smoke-testing it in GH CI

Comment (leogermond) [reply internal_dev_guide.md:21]: nit: the `requirements.txt` is usually lowercase, which allows for eg `find`'ing it or autocomplete without too much thinking

**Review by leogermond — CHANGES_REQUESTED** [review]
Missing requirements.txt

File: internal_dev_guide.md (hunk 1 above)
Comment (pmderodat) [reply internal_dev_guide.md]: Oops, indeed, sorry! All should be fixed now.

**Review by leogermond — APPROVED** [review]
Looks perfect, thanks PM!

**Comment by pmderodat** [comment]
Thank you for the review :-)

**Pull request merged**